src/Components/Dropdown.purs

```diff
@@ -0,0 +1,171 @@
+module Ocelot.Components.Dropdown where
+
+import Prelude
+
+import Data.Array (mapWithIndex)
+import Data.Maybe (Maybe(..), maybe)
+import Effect.Aff.Class (class MonadAff)
+import Halogen as H
+import Halogen.HTML as HH
+import Halogen.HTML.Events as HE
+import Halogen.HTML.Properties as HP
+import Ocelot.Block.Button as Button
+import Ocelot.Block.Icon as Icon
+import Select as Select
+import Select.Utils.Setters (setContainerProps, setItemProps, setToggleProps)
+
+data Query item a
+  = HandleSelect (Select.Message (Query item) item) a
+  | Receive (Input item) a
+  | SetItems (Array item) a
+
+type State item =
+  { isOpen :: Boolean
```

Comment:
isn't this tracked by `Select` already?

https://github.com/citizennet/purescript-halogen-select/blob/master/src/Select.purs#L210

is it necessary to duplicate here?

Reply:
Good call, I'll remove it here.